medical_pathology/demo/medical_pathology_category_demo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="medical_pathology_category_1" model="medical.pathology.category">
```

Comment:
Perhaps we should also specify a proper demo naming scheme as well for next version. I think the general agreement was demo_model_name_#, but don't think it's in docs yet.